scripts/tests/c.sh

```diff
@@ -1,13 +1,42 @@
 STR1_EXEC="docker-compose --ansi never exec -T stride1 strided --home /stride/.strided --chain-id STRIDE"
+GAIA1_EXEC="docker-compose --ansi never exec -T gaia1 gaiad --home /gaia/.gaiad --chain-id GAIA"
 GETBAL() {
     head -n 1 | grep -o -E '[0-9]+'
   }
-IBCATOM='ibc/9117A26BA81E29FA4F78F57DC2BD90CD3D26848101BA880445F119B22A1E254E'
+IBCATOM='ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
```

Comment:
does the setup logic affect this denom? I think this is a hash of source port + source channel, so it's possible initializing channels in a different order could affect this

Reply:
Agreed this is dependent on the ordering in the init file. If we change it, we'll have to change this